src/igvfd/upgrade/file.py

```diff
@@ -57,3 +57,13 @@ def sequence_file_3_4(value, system):
         else:
             notes = f'The mean read length previously exceeded the upper limit of 300000000 so was set to 300000000, please assign the appropriate length.'
         value['notes'] = notes
+
+
+@upgrade_step('reference_file', '4', '5')
+@upgrade_step('matrix_file', '1', '2')
+@upgrade_step('signal_file', '1', '2')
+@upgrade_step('configuration_file', '1', '2')
+@upgrade_step('alignment_file', '1', '2')
+def file_4_5(value, system):
+    if 'dbxrefs' in value:
```

Comment:
should not this step be true ONLY if dbxref is an empty string?

Reply:
these files atp can only have dbxrefs if it consists of empty string(s)